app/src/main/java/com/github/alfabravo2013/readyforexams/presentation/create/CreateViewModel.kt

```diff
@@ -1,6 +1,48 @@
 package com.github.alfabravo2013.readyforexams.presentation.create
 
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.github.alfabravo2013.readyforexams.domain.create.AddTaskUseCase
+import com.github.alfabravo2013.readyforexams.domain.create.CreateChecklistUseCase
+import com.github.alfabravo2013.readyforexams.presentation.models.TaskRepresentation
+import com.github.alfabravo2013.readyforexams.util.Result
+import com.github.alfabravo2013.readyforexams.util.SingleLiveEvent
+import kotlinx.coroutines.launch
 
-class CreateViewModel : ViewModel() {
+class CreateViewModel(
+    private val createChecklistUseCase: CreateChecklistUseCase,
+    private val addTaskUseCase: AddTaskUseCase,
+    private val loadCreatedTasksUseCase: LoadCreatedTasksUseCase
```

Comment:
I would use getCreatedTaskUseCase and rename LoadCreatedTaskUseCase to GetCreatedTaskUseCase